document/src/main/resources/config/liquibase/changelog/20220923-add-assginee-columns-changelog.xml

```diff
@@ -0,0 +1,12 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+    <changeSet author="Rick Veenstra" id="1">
+        <addColumn tableName="json_schema_document">
```

Comment:
The database should also have some kind of an identifier for the assignee. id/email

Reply:
I've added it